src/currentprocess/filesource.rs

```diff
@@ -79,71 +79,38 @@ impl StdinSource for super::TestProcess {
     }
 }
 
-// -------------- stdout -------------------------------
-
-pub trait Isatty {
-    fn isatty(&self) -> bool;
-}
-
-/// Stand-in for std::io::StdoutLock
+/// Allows you to write to a terminal or other output stream.
 pub trait WriterLock: Write {}
 
-/// Stand-in for std::io::Stdout
-pub trait Writer: Write + Isatty + Send {
+/// Holds a terminal or other output stream.
+pub trait Writer: Send + Sync {
+    /// Locks the terminal so that you can write to it.
     fn lock(&self) -> Box<dyn WriterLock + '_>;
 }
 
-/// Stand-in for std::io::stdout
+// -------------- stdout -------------------------------
+
 pub trait StdoutSource {
-    fn stdout(&self) -> Box<dyn Writer>;
+    fn stdout(&self) -> Box<dyn Terminal>;
 }
 
 // -------------- stderr -------------------------------
 
-/// Stand-in for std::io::stderr
```

Comment:
I'm not clear why this docstring is removed either?

Reply:
Revered!